_playwright-tests/Integration/CanInstallUploadRepositryOnHost.spec.ts

```diff
@@ -0,0 +1,135 @@
+import path from 'path';
+import { test, expect, cleanupRepositories, cleanupTemplates, randomName } from 'test-utils';
+import { RHSMClient, refreshSubscriptionManager } from './helpers/rhsmClient';
+import { runCmd } from './helpers/helpers';
+import { navigateToRepositories, navigateToTemplates } from '../UI/helpers/navHelpers';
+import { closePopupsIfExist, getRowByNameOrUrl, retry } from '../UI/helpers/helpers';
+
+const uploadRepoName = 'Upload_Repo';
+test.describe('Install Upload Repositories on Host', () => {
+  test('Install Upload Repositories on Host', async ({ page, client, cleanup }) => {
+    await cleanup.runAndAdd(() => cleanupRepositories(client, uploadRepoName));
+    await closePopupsIfExist(page);
+    await navigateToRepositories(page);
+
+    let regClient: RHSMClient;
+    let templateName: string;
+    let hostname: string;
+    const uploadRepo = uploadRepoName;
+
+    await test.step('Create upload repository', async () => {
+      await page.getByRole('button', { name: 'Add repositories' }).first().click();
+      await expect(page.locator('div[id^="pf-modal-part"]').first()).toBeVisible();
+      const nameInput = page.getByPlaceholder('Enter name');
+      await nameInput.click();
+      await nameInput.fill(uploadRepoName);
+      await page.getByLabel('Upload', { exact: true }).check();
+      await page.getByRole('button', { name: 'filter architecture' }).click();
+      await page.getByRole('menuitem', { name: 'x86_64' }).click();
+      const versionFilterButton = page.getByRole('button', { name: 'filter OS version' });
+      await versionFilterButton.click();
+      await page.getByRole('menuitem', { name: 'el9' }).click();
+      await page.getByRole('menuitem', { name: 'el8' }).click();
+      await versionFilterButton.click(); // Close the filter dropdown
+      const errorElement = page.locator('.pf-v5-c-helper-text__item.pf-m-error');
```

Comment:
this locator is using patternfly 5 which we're no longer using so i don't think we'd ever see this :sweat_smile: is there some error text you could target instead?